install/Knative-custom-install.md

```diff
@@ -145,197 +143,190 @@ service mesh. If you install any of the following options, you must install the
 [2.1]: https://prometheus.io
 [2.2]: https://grafana.com
-[3]: https://github.com/knative/build/releases/tag/v0.2.0
-[3.1]: https://github.com/knative/build/releases/download/v0.2.0/release.yaml
-[4]: https://github.com/knative/eventing/releases/tag/v0.2.1
-[4.1]: https://github.com/knative/eventing/releases/download/v0.2.1/release.yaml
-[4.2]:
-  https://github.com/knative/eventing/releases/download/v0.2.1/eventing.yaml
-[4.3]:
-  https://github.com/knative/eventing/releases/download/v0.2.1/in-memory-channel.yaml
-[4.4]: https://github.com/knative/eventing/releases/download/v0.2.1/kafka.yaml
-[5]: https://github.com/knative/eventing-sources/releases/tag/v0.2.1
-[5.1]:
-  https://github.com/knative/eventing-sources/releases/download/v0.2.1/release.yaml
-[5.2]:
-  https://github.com/knative/eventing-sources/releases/download/v0.2.1/release-with-gcppubsub.yaml
-[5.3]:
-  https://github.com/knative/eventing-sources/releases/download/v0.2.1/message-dumper.yaml
-[6]:
-  https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#event-v1-core
+[2.3]: https://zipkin.io/
+[3]: https://github.com/knative/build/releases/tag/v0.3.0
+[3.1]: https://github.com/knative/build/releases/download/v0.3.0/release.yaml
+[4]: https://github.com/knative/eventing/releases/tag/v0.3.0
+[4.1]: https://github.com/knative/eventing/releases/download/v0.3.0/release.yaml
+[4.2]: https://github.com/knative/eventing/releases/download/v0.3.0/eventing.yaml
+[4.3]: https://github.com/knative/eventing/releases/download/v0.3.0/in-memory-channel.yaml
+[4.4]: https://github.com/knative/eventing/releases/download/v0.3.0/kafka.yaml
+[5]: https://github.com/knative/eventing-sources/releases/tag/v0.3.0
+[5.1]: https://github.com/knative/eventing-sources/releases/download/v0.3.0/release.yaml
+[5.2]: https://github.com/knative/eventing-sources/releases/download/v0.3.0/release-gcppubsub.yaml
+[5.3]: https://github.com/knative/eventing-sources/releases/download/v0.3.0/message-dumper.yaml
+[6]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#event-v1-core
 [6.1]: https://developer.github.com/v3/activity/events/types/
-[6.3]: https://github.com/knative/docs/tree/master/eventing#containersource
+[6.2]: https://github.com/knative/docs/tree/master/eventing#containersource
+[6.3]: https://github.com/knative/eventing-sources/blob/master/samples/cronjob-source/README.md
 [6.4]: https://cloud.google.com/pubsub/
 
-### Installing Knative packages
+### Installing Knative
 
-**Tip**: From the table above, copy and paste the URL and package filename into
-the commands below.
+**Tip**: From the table above, copy and paste the URL and filename into the
+commands below.
 
-1. To install a Knative package by specifying the package's path and filename in
-   the `kubectl apply` command:
+1. To install a Knative component by specifying the filename in the `kubectl apply` command:
 
-   - To install an individual package:
+   - To install an individual component:
```

Comment:
"component" is technically incorrect since you have the observability plugins, channels, and event sources (which are not "components") also listed above as installable options